### ISSUE TITLE
Problem setting dataloader contextToOptions with dataloader-sequelize 2.0.0

### ISSUE THREAD

**Issue by caioflores** [mickhansen/graphql-sequelize#670]
Following this discussion: https://github.com/mickhansen/graphql-sequelize/issues/656

The example server of dataloader with graph-yoga is not working anymore with dataloader-sequelize 2.0.0: https://github.com/mickhansen/graphql-sequelize/blob/master/examples/graphql-yoga/src/server.js

The new way to make it work as @intellix solved is:

```
schema.context = req => {
  const dataloaderContext = createContext(sequelize);

  resolver.contextToOptions = {
    dataloaderContext: [EXPECTED_OPTIONS_KEY]
  };
  return {
    ...req,
    dataloaderContext
  };
};
```

I'll open a PR to update README and example.

**Comment by caioflores** [comment]
Solved the problem here #671

**Comment by mickhansen** [comment]
Will discuss in PR.
